.github/workspace/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test Go
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: 1.23.x
+
+    - name: go mod tidy
+      run: go mod tidy
+
+    - name: go vet
```

Comment:
What does `go vet` do?

Reply:
go vet ([docs](https://pkg.go.dev/cmd/vet))
> Vet examines Go source code and reports suspicious constructs, such as Printf calls whose arguments do not align with the format string. Vet uses heuristics that do not guarantee all reports are genuine problems, but it can find errors not caught by the compilers.

So it should catch stuff like
```go
var answer bool
fmt.Sprintf("true or false: %d", answer)
```

It might prove to be too aggressive, but it's so much easier to start super strict everywhere and then peel back as edge cases are discovered. We'll see how it goes.